service/controller/v2/arm_templates/vmss.json

```diff
@@ -11,6 +11,12 @@
                 "provider": "F80D01C0-7AAC-4440-98F6-5061511962AD"
             }
         },
+        "versionBundleVersion": {
+            "type": "string",
+            "metadata": {
+              "description": "The versionbundle version used to reconcile the guest cluster."
+            }
+        },
```

Comment:
I don't see this passed from the `main.json`.

Reply:
I am not sure what is missing here. How does "passing it" work or look like?

Reply:
Is it just that this block has to be defined the same way in the `main.json`?

Reply:
It has to be passed to the child template like other parameters: https://github.com/giantswarm/azure-operator/blob/fcb47d768c8c9a8b7864f91d8e46b67c2de561bc/service/controller/v1/arm_templates/main.json#L351-L365

Reply:
Can you take a look again?

Reply:
This now looks good 👍